vprof/code_heatmap.py

```diff
@@ -101,7 +101,8 @@ def _skip_lines(src_code, skip_map):
             code_with_skips.extend(
                 ['line', i + j + 1, l] for j, l in enumerate(src_code[i:line]))
             if code_with_skips \
-                    and code_with_skips[-1][0] == 'skip':  # Merge skips.
+                    and code_with_skips[-1][0] == 'skip':  # Is PEP8 conformant
```

Comment:
The comment is not necessary here. I just wanted to point that it's better to use parentheses instead of backslash.